src/services/message.service.ts

```diff
@@ -31,6 +31,46 @@ export class MessageService {
     this._sendConstructedReport(report, { files: message.attachments.map((e) => e.url) });
   }
 
+  async sendMessage(
+    message: IMessage,
+    content: string,
+    reply: boolean,
+    argHeader?: string,
+    argFooter?: string,
+    argBreakChar?: string
+  ): Promise<Boolean> {
+    const header = argHeader ? argHeader + '\n' : '';
+    const footer = argFooter ? '\n' + argFooter : '';
+    const breakChar = argBreakChar || '';
+
+    const replyLength = reply ? ('<@' + message.member?.id + '>,  ').length : 0;
+    const miscLength = header.length + footer.length;
+
+    let messagesToSend: string[] = [header + content + footer];
+
+    if (content.length + miscLength + replyLength > Constants.MaxMessageLength) {
+      if (!breakChar) {
+        return this._failedToSend(message, `Message was over Discords character cap.`);
```

Comment:
we shouldn't be telling the user that we failed to send them the message they want. it should be caught by caller of this function who then must handle it somehow.